src/overlays/actors/ovl_En_An/z_en_an.c

```diff
@@ -13,160 +20,3441 @@
+        }
+    }
+
+    return csId;
+}
+
+s32 EnAn_MsgEvent_ReceiveLetterFromPostman(Actor* thisx, PlayState* play) {
+    EnAn* this = THIS;
+    s16 csId = EnAn_GetCsId(this, 0);
+    s32 ret = false;
+
+    switch (this->msgEventState) {
+        case 0x0:
+            ret = false;
+            if (!EnAn_ChangeCutscene(this, csId)) {
+                break;
+            }
+            FALLTHROUGH;
+
+        case 0x2:
+        case 0x4:
+        case 0x6:
+            Camera_SetTargetActor(Play_GetCamera(play, CutsceneManager_GetCurrentSubCamId(csId)), &this->actor);
+            ret = true;
+            this->msgEventState++;
+            break;
+
+        case 0x1:
+        case 0x3:
+        case 0x5:
+            if ((CHECK_WEEKEVENTREG(WEEKEVENTREG_LISTENED_ANJU_POSTMAN_CONVERSATION)) && (this->msgEventState == 3)) {
+                CutsceneManager_Stop(csId);
+            } else if ((this->actor.child != NULL) && (this->actor.child->update != NULL)) {
+                Camera_SetTargetActor(Play_GetCamera(play, CutsceneManager_GetCurrentSubCamId(csId)),
+                                      this->actor.child);
+            }
+            this->msgEventState++;
+            ret = true;
+            break;
+
+        case 0x7:
+            CutsceneManager_Stop(csId);
+            ret = true;
+            this->msgEventState++;
+            break;
+
+        default:
+            break;
+    }
+
+    return ret;
+}
+
+s32 EnAn_MsgEvent_AttendGoron(Actor* thisx, PlayState* play) {
+    EnAn* this = THIS;
+    s16 csId = EnAn_GetCsId(this, 0);
+    s32 ret = false;
+
+    switch (this->msgEventState) {
+        case 0x0:
+            ret = false;
+            if (!EnAn_ChangeCutscene(this, csId)) {
+                break;
+            }
+            FALLTHROUGH;
+
+        case 0x2:
+        case 0x4:
+            Camera_SetTargetActor(Play_GetCamera(play, CutsceneManager_GetCurrentSubCamId(csId)), &this->actor);
+            ret = true;
+            this->msgEventState++;
+            break;
+
+        case 0x1:
+        case 0x3:
+            if (!(CHECK_WEEKEVENTREG(WEEKEVENTREG_RECEIVED_ROOM_KEY)) && (this->msgEventState == 3)) {
+                CutsceneManager_Stop(csId);
+                this->msgEventState = 5;
+            } else if ((this->actor.child != NULL) && (this->actor.child->update != NULL)) {
+                Camera_SetTargetActor(Play_GetCamera(play, CutsceneManager_GetCurrentSubCamId(csId)),
+                                      this->actor.child);
+            }
+            this->msgEventState++;
+            ret = true;
+            break;
+
+        case 0x5:
+            CutsceneManager_Stop(csId);
+            ret = true;
+            this->msgEventState++;
+            break;
+
+        default:
+            break;
+    }
+
+    return ret;
+}
+
+s32 EnAn_MsgEvent_GiveLunchToGranny(Actor* thisx, PlayState* play) {
+    EnAn* this = THIS;
+    s16 csId = EnAn_GetChildCsId(this, 0);
+    s32 ret = 0;
```

Comment:
```suggestion
    s32 ret = false;
```
This ret can be true/false like the above functions